sdk/maps/Azure.ResourceManager.Maps/api/Azure.ResourceManager.Maps.netstandard2.0.cs

```diff
@@ -102,11 +105,22 @@ public static partial class MapsExtensions
 }
 namespace Azure.ResourceManager.Maps.Models
 {
+    public partial class AccountSasContent
```

Comment:
Consider renaming to `MapsAccountSasConent` to align with the other types, such as `MapsAccountSasToken`.

Reply:
Updated